azure-pipelines.yml

```diff
@@ -203,11 +203,17 @@ jobs:
       versionSpec: '$(python.version)'
 
   - bash: |
-      #!/usr/bin/env bash
       set -ev
-      . scripts/ci/install.sh
-      # verify azdev test works
-      source /home/vsts/work/1/s/env/bin/activate
+
+      # install azdev from source code
+      pip install -e .
+      pip install pytest-cov
```

Comment:
we install it but not use?

Reply:
We can remove this, this was just to see if we could get this job to pass by copying what the Unittest job did since that was passing. This change allowed this job to pass, but we can remove this since yes it is not necessary to passing.

Reply:
"Don't add substance if it is not necessary" pls.